.changes/v3.1.0/590-improvements.md

```diff
@@ -0,0 +1,2 @@
+* Provider: add support for bearer tokens in addition to authorization tokens [GH-590]
+* Provider: enable fallback connection using `/cloudapi/1.0.0/sessions/provider` when `/api/sessions` was disabled [GH-590]
```

Comment:
Here's a small suggestion that may be easier to grasp for less advanced users:

```suggestion
* Provider: automatically use `/cloudapi/1.0.0/sessions/provider` when `/api/sessions` is disabled [GH-590]
```

Reply:
amended